src/data/markdown/docs/02 javascript api/07 k6-experimental/03 websockets.md

```diff
@@ -5,22 +5,42 @@ excerpt: "k6 websockets experimental API"
 
 <ExperimentalBlockquote />
 
-This experimental API implements the browser's [WebSocket API](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket).
-
-With some caveats:
-
-- The [binaryType](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/binaryType) is by default ArrayBuffer and setting it panics as k6 does not support Blob.
-
-
-
-| Class                                       | Description                                                                                    |
-| :------------------------------------------ | :--------------------------------------------------------------------------------------------- |
-| [WebSocket](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/WebSocket)     | Constructs a new WebSocket connection |
+This experimental API implements the browser [WebSocket API](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket) with additional k6-specific functionalities (cookies, tags, headers and so on).
+
+The main difference between this module and [`k6/ws`](/javascript-api/k6-ws/) is that this module uses a global event loop instead of a local one, which enables a single VU to have multiple concurrent connections for better performance.
+
+It's not fully implemented, but we're working on it. If you have any feedback, please [open an issue](https://github.com/grafana/xk6-websockets/) in the extension repository. Our long-term goal is to replace [`k6/ws`](/javascript-api/k6-ws/) module with this one.
```

Comment:
Re "It's not fully implemented", do we have a list of missing features or bugs we're still working on? I.e. is anything preventing users from using it today instead of `k6/ws`? The [currently open issues](https://github.com/grafana/xk6-websockets/issues) don't seem to mention anything major.

We should probably clarify what we mean by "not fully implemented", and link to the open issues (and, eventually, a roadmap), just so users can get a sense of the progress, and when and if they should start using this module instead of `k6/ws`.

Reply:
OK, understood. Let's leave this as is then.